go.mod

```diff
@@ -160,6 +162,7 @@ require (
 
 // Pinned to kubernetes-1.26.2
 replace (
+	github.com/google/cadvisor => github.com/google/cadvisor v0.43.0
```

Comment:
we previously consumed 0.46.0, I'd fix the code to use the newer version (should any change be needed) rather than jumping back

Reply:
I just did, this was the reason I had to fix debug tools first.